src/mappings/platform.ts

```diff
@@ -36,7 +36,7 @@ export function handleUpdate(event: ethereum.Event): void {
     rewardToken.price = getPrice(rewardToken);
     rewardToken.updated = event.block.timestamp;
 
-    updatePricing(geyser, contract, stakingToken, rewardToken, event.block.timestamp);
+    updatePricing(geyser, <Platform>platform, contract, stakingToken, rewardToken, event.block.timestamp);
     geyser.updated = event.block.timestamp;
 
     // store
```

Comment:
I think we need a `platform.save();` at the end of this function... that's something I missed earlier

Reply:
oh also... I missed a string param slot in that "Marking geyser as stale" log message below. could you include that here?